docs/source/user_guide/interoperability.rst

```diff
@@ -155,20 +155,16 @@ PyTorch also supports zero-copy data exchange through ``DLPack`` (see :ref:`dlpa
 	import cupy
 	import torch
 
-	from torch.utils.dlpack import to_dlpack
 	from torch.utils.dlpack import from_dlpack
```

Comment:
PyTorch is working toward array-api compliance, and this function is already available in the main namespace.
```suggestion
	from torch import from_dlpack
```

Reply:
Sounds better!